packages/wrangler/src/config/environment.ts

```diff
@@ -210,7 +218,7 @@ interface EnvironmentInheritable {
 	 * Refer to the [custom builds documentation](https://developers.cloudflare.com/workers/cli-wrangler/configuration#build)
 	 * for more details.
 	 *
-	 * @default {}
+	 * @default {watch_dir:"./src"}
```

Comment:
I don't think this is accurate? AFAIK the default for `build.watch_dir` is `undefined` (or the current working directory if `build.command` is defined)

Reply:
is [this test](https://github.com/cloudflare/workers-sdk/blob/main/packages/wrangler/src/__tests__/configuration.test.ts#L23) incorrect then?

Reply:
Oh sorry, yeah, looks like I was reading the code wrong—apologies!